### ISSUE TITLE
Application slowness/not processing new requests

### ISSUE THREAD

**Issue by slamkalukas** [Activiti/Activiti#2700]
Hello team,

we got running Activiti 7 Cloud Full example with custom runtime bundle, cloud connectors and query service. 

We observe an application slowness and issue with closing tasks with OOTB REST APi after 6/7 days of running our pods. In logs we see issue - RabbitMQ broken pipe error.

This is second time we observe this problem, last time application was slow (long waiting for REST APIs) but some time we were unable to get list of Tasks (via REST API, list should consist of 10-15 items). Sometime we also observe this when we are redeploying some pods - this time APIs are not accessible immediately.

In first approximately 1-4 days, everything is looking good, no issues, no slowness.

Have you seen similar behaviour before?

**Comment by salaboy** [comment]
@slamkalukas thanks a lot for reporting this. 
When you say: OOTB REST APi  does that mean RuntimeBundle/v1/tasks/complete ?

**Comment by slamkalukas** [comment]
@salaboy yes, runtimeBundle/v1/tasks/taskId/complete and also in our customRuntimeBundle with our new processes

**Comment by salaboy** [comment]
Also.. which database are you using? and are you using Rabbit with any other configuration? 
Rabbit is not setup for HA.. so you might want to tweak the full example to have a more robust setup

**Comment by salaboy** [comment]
@slamkalukas and if you do query/v1/tasks how many tasks do you have?

**Comment by salaboy** [comment]
or even your GET runtimebundle/v1/tasks/ how many tasks do you have?

**Comment by slamkalukas** [comment]
We are using Microsoft SQL, we got maximum 20 process instances running with maximum 40 tasks

**Comment by salaboy** [comment]
@slamkalukas can you use MSSQL admin console to evaluate the query speeds? Or is it clear in your environment that RabbitMQ is the problem?

**Comment by salaboy** [comment]
@slamkalukas I would recommend you to take a look at: https://github.com/helm/charts/tree/master/stable/rabbitmq-ha
and please report back about your findings, we will start some work around the topic but from the community perspective there is much that we can do.. so for example testing with MSSQL Server is not going to happen soon.
